core/create.go

```diff
@@ -125,13 +133,28 @@ func CreateTheme(path, name string) error {
 }
 
 // CreateFile creates a file with specified path under content directory.
-func CreateFile(filePath string) error {
+func CreateFile(filePath string, options CreateFileOptions) error {
+
+	if _, err := os.Stat(options.Project); os.IsNotExist(err) {
+		return ErrProjectNotExists
+	}
 
-	contentPath := filepath.Join(ContentDir, filePath)
+	var contentPath string
+	if options.Project != "." {
+		contentPath = filepath.Join(options.Project, ContentDir, filePath)
+
+	} else {
+		contentPath = filepath.Join(ContentDir, filePath)
+	}
```

Comment:
Does simplifying this to `contentPath := filepath.Join(options.Project, ContentDir, filePath)` (because `options.Project` defaults to `.`) work as well?